go-hello-nonroot/README.md

```diff
@@ -0,0 +1,20 @@
+# Go webapp that returns the hostname, IP address, the request URI, the local time and the system user running the app in the server.
+
+The image is uploaded to Docker Hub -- https://hub.docker.com/r/nginxdemos/go-hello/.
+
+How to run:
+```
+$ docker run -P -d nginxdemos/go-hello
```

Comment:
I think it's better to run explicitly on port 8080 so we don't have to look up what port it's running on. And better to run interactively by default.

```suggestion
$ docker run -p 8080 nginxdemos/go-hello
```